src/bin/pg_autoctl/cli_common.c

```diff
@@ -309,6 +317,81 @@ cli_create_node_getopts(int argc, char **argv,
 				break;
 			}
 
+			case 's':
+			{
+				/* { "ssl-self-signed", no_argument, NULL, 's' }, */
+				if (sslOrNoSSLCount > 0)
```

Comment:
we can get into this state with having more than one -s. Error is not really accurate in that way

Reply:
Good point. I stopped being lazy about it and introduced an enum type which allows for better tracking of the user choice here.